api/api-doc/messaging_service.json

```diff
@@ -249,7 +249,7 @@
                  "MIGRATION_REQUEST",
                  "PREPARE_MESSAGE",
                  "PREPARE_DONE_MESSAGE",
-                 "STREAM_MUTATION",
+                 "UNUSED_STREAM_MUTATION",
```

Comment:
This reads like a legit verb name, maybe separate `UNUSED` with two `_`, like: `UNUSED__STREAM_MTUATION`.

Reply:
Ok.